src/Symfony/Component/Routing/Requirement/Requirement.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Routing\Requirement;
+
+/**
+ * A collection of universal regular-expression constants to use as route parameter requirements.
+ */
+enum Requirement
+{
+    public const ASCII_DASH_SLUG = '[A-Za-z0-9]+(?:-[A-Za-z0-9]+)?'; // symfony/string AsciiSlugger default implementation
```

Comment:
Given that "slug" means "with dashes" most of the times, maybe we can rename this as just `ASCII_SLUG`? Also, many times slugs don't include dashes because the original contents didn't contain spaces, so "DASH" might confuse people about being mandatory to have at least one dash.